### PR TITLE
localproxy mode for edgemicro

### PR DESCRIPTION
**Pull request by mukundha**
I added localproxy support for micro sometime back - so it can run without having to pull config from edge. this was helpful when embedding microgateway as a sidecar container when deploying applications in containers [specifically w/ kubernetes], 
do you think this will be useful for mainstream micro usecases?
This is supposed to work with this PR - https://github.com/apigee/microgateway-config/pull/6

### CODE REVIEW

**Comment by mdobson** [comment]
Any contribution is useful :) I'll take a look.

**Comment by srinandan** [comment]
I think this is a very useful feature (to load selective proxies). Has been asked by customers many times.

File: cli/lib/gateway.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ Gateway.prototype.start = function start(options, cb) {
   const args = { target: cache, keys: keys,pluginDir:options.pluginDir };
   const that = this;
   if (cluster.isMaster) {
-    edgeconfig.get({ source: source, keys: keys }, function (err, config) {
+    edgeconfig.get({ source: source, keys: keys, localproxy: {name:options.app_name , path:options.app_path, target_url:options.target_url} }, function (err, config) {
```
Comment (F1ERRO) [inline cli/lib/gateway.js]: Won't this cause your conditional here to always be true, always preferring localproxy?

https://github.com/apigee/microgateway-config/pull/6/commits/11f77a619a840cdca59bb3dc7ecea037435d136a#diff-9a3ce09856114605620e98d815bd93e1R56

**Comment by WWitman** [comment]
If we're calling the feature "local proxies", it might be more clear to call the config parameters proxy_name and proxy_path, instead of app_name and app_path.

**Pull request closed without merging**